libslide/src/emit.rs

```diff
@@ -187,7 +187,11 @@ impl Emit for Expr {
 
     fn emit_latex(&self) -> String {
         match self {
-            Self::Const(num) => num.to_string(),
+            Self::Const(num) => match num.to_string().as_ref() {
+                "inf" => "\\infty",
+                other => other,
```

Comment:
why double \\

Reply:
`\` is an escape character, so we have to use `\\` to escape it (it becomes `\`)